### DISCUSSION TITLE
Blog sample data: Why language tag in title?

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#33859] [Q&A] [answered]
In an multilanguage site user can install blog sample data for each active language in Backend. This is fine.

The language tag is appended in asset-titles causing effects:

![image](https://user-images.githubusercontent.com/78906980/118237403-95219700-b497-11eb-8b90-073a6d16e083.png)
![image](https://user-images.githubusercontent.com/78906980/118237464-a1a5ef80-b497-11eb-89d7-1274d986600b.png)
![image](https://user-images.githubusercontent.com/78906980/118237570-b7b3b000-b497-11eb-99b2-dffc6806fa08.png)
![image](https://user-images.githubusercontent.com/78906980/118237593-bf735480-b497-11eb-80bc-911b58db9517.png)

I don't find the reason to append the tag in title, cause the assets are assigned to languages. I prefer to avoid lang-tags in title.

**Comment by brianteeman** [comment]
It is purely for demonstration purposes

**Reply by ghost** [reply]
The demonstration purpose is: The asset is assigned to a language? 

The Language-Switcher and the different Language-Values (Search, Suche, Recherche, …) aren't enough for a frontend-user?

**Reply by richard67** [reply]
It's necessary for the aliases, not for the titles, but due to the way how blog sampledata installation works this might be the reason, because aliases are generated from the titles.

**Reply by brianteeman** [reply]
Nice idea but not correct. the alias is generated by the script to specifically include the language tag. No real reason or benefit other than as a helper when learning the different parts of a multilingual site
`		$alias = 'allcategories_' . $itemLanguage->language;
`

**Reply by ghost** [reply]
I guess the reason is the "alias", which is handled different by components having different languages.

Example article: Same alias possible because every article is set to a different language

> ![image](https://user-images.githubusercontent.com/78906980/118349786-23a71e80-b553-11eb-8f76-575777089cbd.png)

Same works for menu-items on same level, works not for Categories, Tags.

**Comment by infograf768** [comment, marked as answer]
There is a good reason.
Some languages may not have in their packs the blog sample data ini file.
In that case, the blog sample data uses the default en-GB ini and, in the manager, the title may be the same as in English.
Best example concerns the German J3 pack. Here a screenshot in J3, but same in J4 as we can atm install only the J3 pack.

<img width="1030" alt="Screen Shot 2021-05-15 at 08 31 48" src="https://user-images.githubusercontent.com/869724/118350630-27896f80-b558-11eb-9a0d-c0af65c9aa9a.png">

and the English

<img width="615" alt="Screen Shot 2021-05-15 at 08 34 52" src="https://user-images.githubusercontent.com/869724/118350702-84852580-b558-11eb-86e6-1edbf13eaa1a.png">

**Comment by ghost** [comment]
Thanks all for your answers, i understand a little bit more :-)
